### PR TITLE
Allow interpolation of title

### PR DESCRIPTION
**Pull request by ojongerius**
Use case: I like having generic, re-usable experiments for different services. Configuration currently does a great job doing that, but I'd love the title to include my service name too.

### CODE REVIEW

File: chaoslib/experiment.py [hunk 1]
```diff
@@ -193,6 +187,13 @@ def run_experiment(experiment: Experiment,
     initialize_controls(experiment, config, secrets)
     activity_pool, rollback_pool = get_background_pools(experiment)
 
+    experiment["title"] = substitute(experiment["title"], config, secrets)
+    logger.info("Running experiment: {t}".format(t=experiment["title"]))
+
+    dry = experiment.get("dry", False)
+    if dry:
+        logger.warning("Dry mode enabled")
+
```
Comment (Lawouach) [inline chaoslib/experiment.py]: Looks good. Why has it moved places however?

Comment (ojongerius) [reply chaoslib/experiment.py]: I needed the loaded config (`config = load_configuration(experiment.get("configuration", {}))`) to be available as input to `substitute()`.

I moved the logging statement about "dry" down to keep the order of log statement as it currently is. If you prefer I could move the dry logic where it was.

Comment (Lawouach) [reply chaoslib/experiment.py]: Yes please, if you don't mind keeping the dry logic where it was. Makes the change more focused :)

**Comment by ojongerius** [comment]
@Lawouach good to merge?

**Comment by Lawouach** [comment]
Oops, yes :)

**Pull request merged**